code-n-color.html

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+
+title: Code'n'Color
+
+author: Lukas
+content_authors: Code'n'Color
+
+last_update: 2020-06-27
+---
+
+<div class="row pb-2 mb-2">
+
+  <div class="col-md-8">  
+    <h1>Code'n'Color</h1>
+    <h4 class="text-secondary">PhD student support group for Black, Indigenous, and other People of Color (BIPOC)</h4>
+  </div>
+
+  <div class="border-bottom col-md-12">  
+  </div>
+</div>
+
+<div class="row mt-2">
+  <div class="col-md-8">
+    <h2>Description</h2>
+    <p>
+      The support group meets weekly for coffee breaks and hosts BIPOC PhD holders who are leaders in the computing industry and academia. Advised and supported by BIPOC faculty in CS, ECE, and LS, the group is intended to provide a safe and supportive space for young scientists to interact with future faculty peers and climb the treacherous slopes of the ivory tower, cognizant of the unique, different demands and obstacles that BIPOC PhD students face.
```

Comment:
Some line breaks could help formatting this paragraph

Reply:
In the displayed paragraph or the source?

Reply:
Hmm ok, I'll add a break between the sentences. But I think they envisioned it as a single paragraph?

Reply:
Fair point